returnn/dataset.py

```diff
@@ -0,0 +1,90 @@
+__all__ = ["ExtractDatasetStatisticsJob"]
+
+from sisyphus import *
+
+import numpy
+import os
+import shutil
+import subprocess
+
+from i6_core.returnn.config import ReturnnConfig
+from i6_core.util import create_executable
+
+
+class ExtractDatasetStatisticsJob(Job):
```

Comment:
If job only computes mean + std-dev it should be called like that. Current name is a bit confusing as I would expect Dataset statistics to be things like #number of sequences, average length, etc.

Reply:
Ok